SECURITY.md

```diff
@@ -1,9 +1,22 @@
 # Security Policy
 
-## Reporting a vulnerability
+## Reporting Security Issues
 
-If you found a vulnerability in Neutralinojs components, please report as a Github [issue](https://github.com/neutralinojs/neutralinojs/issues).
```

Comment:
Hey.. I am planning to add a page about security to the documentation soon. Therefore, could you please submit this PR without the `SECURITY.md` file-related changes?

Reply:
Okay, no problem